### PR TITLE
App strings reworked

### PR DESCRIPTION
**Pull request by comradekingu**

### CODE REVIEW

File: src/main/res/values/strings.xml [hunk 1]
```diff
@@ -3,14 +3,14 @@
     <string name="action_failed">Action failed</string>
     <string name="add_repository">Add repository</string>
     <string name="address">Address</string>
-    <string name="all_applications">All Applications</string>
-    <string name="all_applications_up_to_date">All Applications up to date</string>
+    <string name="all_applications">All applications</string>
+    <string name="all_applications_up_to_date">All your applications are up to date</string>
     <string name="already_exists">Already exists</string>
     <string name="always">Always</string>
     <string name="amoled">Amoled</string>
```
Comment (comradekingu) [inline src/main/res/values/strings.xml:10]:
```suggestion
    <string name="amoled">Black</string>
```
Should be AMOLED, but more correct to call it "Black" if it is the theme setting.

File: src/main/res/values/strings.xml [hunk 2]
```diff
@@ -45,11 +45,11 @@
...
     <string name="http_proxy">HTTP proxy</string>
-    <string name="ignore_all_updates">Ignore all updates</string>
-    <string name="ignore_this_update">Ignore this update</string>
+    <string name="ignore_all_updates">Ignore all new versions</string>
+    <string name="ignore_this_update">Ignore this version</string>
     <string name="incompatible_api_DESC_FORMAT">Your %1$s (API version %2$d) is not supported. %3$s</string>
-    <string name="incompatible_api_max_DESC_FORMAT">Maximum API version is %d.</string>
-    <string name="incompatible_api_min_DESC_FORMAT">Minimum API version is %d.</string>
+    <string name="incompatible_api_max_DESC_FORMAT">The max. API version is %d.</string>
+    <string name="incompatible_api_min_DESC_FORMAT">The min. API version is %d.</string>
```
Comment (NoahAndrews) [inline src/main/res/values/strings.xml:52]: Having periods after max and min feels very wrong.

File: src/main/res/values/strings.xml [hunk 3]
```diff
@@ -161,12 +161,12 @@
...
     <string name="update">Update</string>
     <string name="updates">Updates</string>
-    <string name="upstream_source_code_is_not_free">Upstream source code is not free</string>
+    <string name="upstream_source_code_is_not_free">The upstream source code is not free</string>
     <string name="username">Username</string>
     <string name="username_missing">Username missing</string>
     <string name="validation_index_error_DESC">Index could not be validated.</string>
     <string name="version">Version</string>
-    <string name="version_FORMAT">Version %s</string>
+    <string name="version_FORMAT">%s version</string>
```
Comment (NoahAndrews) [inline src/main/res/values/strings.xml:169]: Are you proposing changing "version 1.2" to "1.2 version"? If so, that is incorrect.

File: src/main/res/values/strings.xml (hunk 2 above)
Comment (comradekingu) [reply src/main/res/values/strings.xml:52]: Sorry, looked that up in the wrong language.

File: src/main/res/values/strings.xml (hunk 3 above)
Comment (comradekingu) [reply src/main/res/values/strings.xml:169]: I don't know what happened there. Good catch :)

**Pull request merged**